ddtrace/contrib/responder/patch.py

```diff
@@ -0,0 +1,109 @@
+
+# stdlib
+
+# project
+import ddtrace
+from ddtrace.ext import http
+from ddtrace.vendor.wrapt import wrap_function_wrapper as _w
+from ddtrace.utils.wrappers import unwrap as _u
+from ddtrace.propagation.http import extract_context_from_http_headers
+
+# 3p
+import responder
+patches = []  # will be overriden when patched funcs are defined.
+
+
+def patch():
+    if _is_patched(responder):
+        return
+    _set_patched(responder, True)
+
+    for obj, mname, fname, func in patches:
+        _w(mname, fname, func)
+
+
+def unpatch():
+    if not _is_patched(responder):
+        return
+    _set_patched(responder, False)
+
+    for obj, _, prop, _ in patches:
+        if '.' in prop:
```

Comment:
Why is this necessary?

Reply:
I suspect it's possible its not there? copypasta from another integration

Reply:
Right, I see it in the flask integration. I see the benefit of its use there being that we the functions patched differ across library versions.

With a fixed number of patched functions as here, a simpler solution might make more sense:

```
# in patch()
_w(responder.api.API, '__call__', _api_call)
_w(responder.api.API, 'template', _api_template)
_w(responder.api.API, 'template', _api_template_string)
_w(responder.routes.Route, '__call__', _route_call)

# in unpatch()
_u(responder.api.API, '__call__')
_u(responder.api.API, 'template')
_u(responder.api.API, 'template')
_u(responder.routes.Route, '__call__')
```